README.md

```diff
@@ -17,19 +17,19 @@ Table of Contents
     * [License](#license)
 
 ## Usage
-Add it as an npm dev-dependency
+Add it as an npm devDependency, as well as all the necessary [peerDependencies](https://nodejs.org/en/blog/npm/peer-dependencies/).
 
 ```bash
-$ npm i eslint-config-commercetools --save-dev
+$ npm install --save-dev eslint-config-commercetools eslint@^#.#.# eslint-config-airbnb@^#.#.# eslint-plugin-jsx-a11y@^#.#.# eslint-plugin-import@^#.#.# eslint-plugin-react@^#.#.#
```

Comment:
Yes we could look how other projects document this, there are [a lot of projects](https://github.com/search?utf8=%E2%9C%93&q=%22npm+i+-g+npm%22+filename%3A.travis.yml&type=Code&ref=searchresults) doing it but not mentioning at all.